sumologic/resource_sumologic_monitors_library_monitor.go

```diff
@@ -611,6 +619,37 @@ var sloBurnRateTriggerConditionSchema = map[string]*schema.Schema{
 	}, sloBurnRateConditionCriticalOrWarningAtleastOneKeys),
 }
 
+var logsAnomalyTriggerConditionSchema = map[string]*schema.Schema{
+	"field": {
+		Type:     schema.TypeString,
+		Required: true,
+	},
+	"direction": {
+		Type:         schema.TypeString,
+		Optional:     true,
+		Default:      "Both",
+		ValidateFunc: validation.StringInSlice([]string{"Both", "Up", "Down"}, false),
+	},
+	"anomaly_detector_type": {
+		Type:         schema.TypeString,
+		Required:     true,
+		ValidateFunc: validation.StringInSlice([]string{"Cluster"}, false),
+	},
+	"sensitivity": {
```

Comment:
shouldn't sensitivity have a validate func which fails when user sets the value > 1 ?

Reply:
shouldn't sensitivity also be under critical ?

Reply:
>shouldn't sensitivity have a validate func which fails when user sets the value > 1 ?

hmm, good catch. We are missing this check from API too. We should add it there.

Reply:
>shouldn't sensitivity also be under critical ?

`direction`, `anomaly_detector_type`, and `sensitivity` are all tied to anomaly model rather than critical trigger. That's why I did not include them under `critical` object.

Reply:
can you confirm with Ash/ Ryley once about sensitivity because that is part of inference evaluation if I'm not wrong.

Reply:
I am going to move `sensitivity` to `critical` trigger.